README.md

```diff
@@ -58,12 +58,11 @@ import { API } from "@stoplight/elements";
 
 <API
   apiDescriptionUrl="https://api.apis.guru/v2/specs/github.com/1.1.4/openapi.yaml"
-  layout="traditional"
```

Comment:
Although there is no `traditional` layout in the `API` component, we still have `sidebar | stacked`

Another thing is that we don't have any description of `StoplightProject` in the main README yet. Do you think we should add it there?

Reply:
Nah, we push people to docs and docs takes care of that.